example_api/main.cpp

```diff
@@ -80,11 +80,11 @@ int main()
 	DumpHex(ttdcursor.GetThreadInfo(), 0x20);
 
 	std::cout << "\nContext:\n";
-	TTD::TTD_Replay_RegisterContext* ctxt = ttdcursor.GetCrossPlatformContext();
+	auto ctxt = ttdcursor.GetContextx86();
```

Comment:
```suggestion
        // Use  ttdcursor.GetContextx86() for x86 context
        auto ctxt = ttdcursor.GetCrossPlatformContext();
```
Is it OK for you tu use comment for x86 illustrations for now?